### PR TITLE
YouTube tutorial for getting started with Fiber

### PR DESCRIPTION
**Pull request by rajee-a**
This is a quick getting started guide with detailed explanations.

### CODE REVIEW

**Comment by efectn** [comment]
Can you add a section for videos? (after articles)

**Comment by rajee-a** [comment]
> Can you add a section for videos? (after articles)
Yes

File: README.md [hunk 1]
```diff
@@ -140,6 +140,11 @@ Articles about Fiber written by the community.
 - [Welcome to Fiber — an Express.js styled web framework written in Go with ❤️](https://dev.to/koddr/welcome-to-fiber-an-express-js-styled-fastest-web-framework-written-with-on-golang-497)
 - [Blazing Fast Unit Tests - Fiber/fasthttp/http Internals](https://medium.com/trendyol-tech/golang-blazing-fast-unit-tests-fiber-fasthttp-http-internals-and-optimizing-http-server-tests-bbd1fe7b944b)
 
+## 📺 Videos
```
Comment (efectn) [inline README.md:145]: Update toc please

**Review by efectn — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (rajee-a) [reply README.md:145]: Done 👍

**Pull request merged**